docs/UserGuide.md

```diff
@@ -121,94 +121,108 @@ who has 6 years of experience and specializing in 'braces' into the list of dent
-**Format:** `search-dentist NAME`
-
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-To check the Dentist ID for a dentist, you can simply enter the command `list-dentist`.
+To check the attributes that you can filter by, you can simply enter the command `list-dentist`.
 </div>
 
-### Deleting a dentist: `delete-dentist DENTIST_ID`
+**Format:** `filter-dentist [a/ATTRIBUTE] [k/KEYWORDS]`
 
-Deletes the specified dentist from ToothTracker.
+Examples:
 
-**Format:** `delete-dentist DENTIST_ID`
+* `filter-dentist a/phone k/98225677` searches for the dentist with the phone number 98225677.
 
-<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-This command is DESTRUCTIVE!
-Dentists deleted will need to be added back and their previous records may be removed.
-Proceed with caution!
-</div>
+### Editing a dentist: `edit-dentist`
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-To check the Dentist ID for a dentist, you can simply enter the command `list-dentist`.
-</div>
+Edits one or more details of the dentist at the specified Dentist ID. This is helpful when:
+* The particulars of a dentist need to be updated.
+* Optional dentist details were not provided previously.
+* You accidentally entered incorrect information about a dentist into the ToothTrack database.
+
+**Format:** `edit-dentist DENTIST_ID [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [s/SPECIALIZATION] [y/YOE] [t/TAG]…​`
 
 Examples:
 
-* `delete-dentist 2` deletes dentist with DENTIST_ID 2 from ToothTracker.
+* `edit-dentist 1 p/98987676 e/bobjuly@gmail.com` edits the phone number and email of the dentist with Dentist ID of 1
+  into 98987676 and bobjuly@gmail.com respectively.
 
-### Listing all dentists : `list-dentist`
 
-Shows a list of all dentists in ToothTracker. This is useful when:
+* `edit-dentist 2 n/Emmanuel Alexandra t/` edits the name of the dentist with Dentist ID of 2 into ‘Emmanual Alexandra’
+  and removes all tags of the dentist.
 
-* You want to retrieve the information of all dentists.
-* You want to verify a dentist is added successfully in ToothTracker.
-* You want to verify a dentist is updated successfully in ToothTracker.
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+You can edit the particulars in any order and you can edit more than one detail of the dentist.
+</div>
 
-**Format:** `list-dentist` (No extra parameters required)
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+To check the Dentist ID for a dentist, you can simply enter the command `list-dentist`.
+</div>
 
+<div markdown="block" class="alert alert-info">
+**:information_source: Notes about editing tags:**<br>
+When editing tags, you have to include any previous tags that was already included in the dentist, or else these tags will be removed. 
+This also means that you can use `t/` to remove all tags from a dentist using the <code>edit-dentist</code> command.
+</div>
 
 ## Part 2 - Patient Features
```

Comment:
1. Under Examples: command should be in lower caps `add-patient`
2. Can double check the order of the parameters for the format of `add-patient`? I think we should standardise with the order we give in the example.